src/styles/utils/vars.scss

```
@@ -0,0 +1,12 @@
+$main-font: Roboto, sans-serif;
+$hover-color: #00acdc;
+$nav-link-color: #060b35;
+$main-accent: #060b35;
```

Why did you create two variables with the same values?
```suggestion
$nav-link-color: #060b35;
```